install/helm/values.yaml

```diff
@@ -190,3 +190,62 @@ configuration:
   cors:
     allowedOrigins:
       - "https://localhost:3000"
+
+# Setup Job configuration
+# Runs setup.sh as a one-time initialization job during Helm install
+setup:
+  # Enable setup job (runs on first install only via Helm hook)
+  enabled: true
```

Comment:
Shall we set the default to false? Otherwise, users will need to override it during Helm upgrade commands.